translate/asm.go

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2018 De-labtory
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package translate
+
+import "github.com/DE-labtory/koa/opcode"
+
+type output struct {
```

Comment:
1. `bytecode` means real raw byte code which you already know. (ex 0x1234...) And `asmcode` means assemble code like `Push 0x01 Pop ...`. It is necessary because of `Jump`. It is also user-friendly type too.

2. `pc` would be used in calculating next branch. `Jump` changes execution flow, so it needs to calculate next or return branch position.